flask_api/app/helpers/generate_trees.py

```diff
@@ -23,6 +23,8 @@ def __init__(
         self.corner_coords = self._coords_to_utm(corner_coords)
         self.distance_between_rows = distance_between_rows
         self.distance_between_trees = distance_between_trees
+        self.number_of_rows = None
+        self.number_of_trees_in_each_row = None
```

Comment:
Consider
```suggestion
        self.number_of_rows: Optional[int] = None
        self.number_of_trees_in_each_row: Optional[int] = None
```
for `None`-safety